src/locales/en/pages.yaml

```diff
@@ -542,6 +602,85 @@ daan:
+    de Delftse Opera Compagnie vertelde dat van alle faciliteiten gebruik gemaakt kon worden om de
+    gemeente en andere gasten in stijl te ontvangen.
+
+    Van veel concerten in het Techniek Museum werden door Rien geluidsopnamen gemaakt. Hij had daar
+    ervaring mee; hij had al diverse plaatopnamen op zijn naam staan (zoals voor het Erasmus label) en vele
+    live‐opnamen (bijvoorbeeld van het Zenith Festival in de Paleiskerk Den Haag). Sinds 2008 hebben we met
+    de KRAK soms apparatuur van hem kunnen lenen, als er specifieke aanvulling nodig was voor het opnemen
+    van Krashna‐concerten.
+
+    De kamermuziekconcerten verhuisden naar het nieuwe Cultureel Centrum van de TU. In dat Centrum gaf
+    Rien overigens ook cursussen, onder andere over 'de opera', die natuurlijk ook door Krashna‐leden gevolgd
+    werden.
+
+    Ook bij de geënsceneerde Mattheuspassie in 1998 bleek Rien onmisbaar. Allerlei zaken als verlengsnoeren,
+    stekkerdozen, een televisie en meer werden door hem aangesleept. Volgens de toenmalige voorzitter van
+    Krashna "een echte redding".
+
+    Door de jaren heen heeft Rien vele activiteiten van Krashna met donaties ondersteund. Hij wilde daar geen
+    bedankjes voor. Hij nam overigens geen blad voor de mond, als hij op‐ of aanmerkingen had. Als
+    weerwoord terecht was, accepteerde hij dat echter ook; de gulle en loyale mopperpot.
+
+    Hij heeft vele concerten van Krashna bezocht, mede als recensent en later als liefhebber en erelid. Soms
+    ging hij zelfs mee op tournee en genoot dan met volle teugen (Aken 2003: "Drie heerlijke dagen waren
+    dat"). De laatste jaren werd hij helaas slechter ter been, waardoor hij niet alle concerten meer kon
+    bezoeken. Het laatste concert van Krashna met Daan Admiraal heeft hij moeten missen. Dat deed hem veel
+    verdriet; hij was erg op Daan gesteld en Daan stond als musicus bij Rien in hoog aanzien.
+
+    Het eerste concert met Stijn Berkouwer, de opvolger van Daan, heeft hij wel kunnen bezoeken (Nederlands
+    Programma). Hij was zeer verheugd over het resultaat en stak dat niet onder stoelen of banken. Dit was zijn
+    laatste concertbezoek van koor‐ en orkest. De tourneeconcerten, de streams en het najaarsconcert 2021
+    heeft hij moeten missen. Zijn laatste kamermuziekconcert was dat van november 2019, waar hij met veel
+    plezier sinterklaasliedjes meezong.
+
+    Ondergetekenden hebben na de zomer nog wat dingen voor Rien kunnen doen; 15 november hebben we
+    hem nog gezien. We hadden de gedachte dat het mooi zou zijn als we hem vanuit Krashna zouden onthalen
+    op concertjes bij hem thuis, waar hij een mooie vleugel had staan. Helaas hebben we het niet meer kunnen
+    verwezenlijken. Het ging te snel. Op 23 november is Rien per ambulance naar een hospice gebracht en is
+    daar op 24 november overleden. 1 december is hij in besloten kring gecremeerd.
+
+    Laten we bij onze uitvoering van het Requiem van Verdi aan Rien denken.
```

Comment:
Maybe remove this line, as it is not exactly relevant for the website

Reply:
I'll have to ask Pepo, he's kinda hypercritical when it comes to these things